src/services/users/types/userResponseTypes.ts

```diff
@@ -0,0 +1,50 @@
+export interface LoginResponse {
```

Comment:
Inconsistent variable naming, some are snake case some are camel case

Reply:
@jherasymuik-ehelply I think Wilson is correct here. These names would come from the Pydantic models that were created in users for the endpoint payloads and endpoint responses.

Reply:
Makes sense! I'll close this comment